CHANGELOG.md

```diff
@@ -4,6 +4,60 @@
 
 # JupyterLab Changelog
 
+## v4.2
+
+### Workspaces UI
+
+It is now possible to switch and manage [workspaces](https://jupyterlab.readthedocs.io/en/latest/user/workspaces.html) from within JupyterLab.
+
+<img alt="A context menu opened over the sidebar with workspaces list. The menu has options to manager the workspace over which it was opened - clone, rename, reset delete, export and import." src="https://raw.githubusercontent.com/krassowski/jupyterlab/ufc4.2/docs/source/getting_started/changelog_assets/4.2-workspaces-sidebar.png" class="jp-screenshot">
```

Comment:
```suggestion
<img alt="A context menu opened over the sidebar with workspaces list. The menu has options to managerthe workspace over which it was opened: clone, rename, reset, delete, export, and import." src="https://raw.githubusercontent.com/krassowski/jupyterlab/ufc4.2/docs/source/getting_started/changelog_assets/4.2-workspaces-sidebar.png" class="jp-screenshot">
```

Copy edit on alt text — thank you for adding this!

Reply:
`managerthe` guess you meant to remove "r", not the space - I will edit you comment

Reply:
Haha, nope there can be no `:` - this is what I had initially but `myst` errors out if there is a colon in `alt`, see my earlier commit https://github.com/jupyterlab/jupyterlab/pull/16103/commits/0fcad9dc7f6f0e4cfb85b9954fe55ab7bc788899. I will edit to remove the colon again.